src/types/account-manager.ts

```diff
@@ -0,0 +1,253 @@
+import algosdk from 'algosdk'
+import {
+  getAccountAssetInformation,
+  getAccountInformation,
+  mnemonicAccountFromEnvironment,
+  multisigAccount,
+  randomAccount,
+  rekeyedAccount,
+} from '../account/account'
+import { getDispenserAccount } from '../account/get-dispenser-account'
+import { mnemonicAccount } from '../account/mnemonic-account'
+import { getKmdWalletAccount } from '../localnet/get-kmd-wallet-account'
+import { getLocalNetDispenserAccount } from '../localnet/get-localnet-dispenser-account'
+import { getSenderAddress, getSenderTransactionSigner } from '../transaction/transaction'
+import { AccountInformation, SigningAccount, TransactionSignerAccount } from './account'
+import { AlgoAmount } from './amount'
+import { ClientManager } from './client-manager'
+import { SendTransactionFrom } from './transaction'
+
+/** Creates and keeps track of signing accounts against sending addresses. */
+export class AccountManager {
+  private _clientManager: ClientManager
+  private _accounts: { [address: string]: TransactionSignerAccount } = {}
+  private _defaultSigner?: algosdk.TransactionSigner
+
+  /**
+   * Create a new account creator
+   * @param algod The algod client to use
```

Comment:
docs don't match the code

Reply:
Fixed in 05bb21ecb117500bc87be6fe2dd5d831e8ad413f